### PR TITLE
Filled out IRecordInfo interface

### PR DESCRIPTION
**Pull request by ZerothLaw**
Noticed this type wasn't defined, so I filled it in. I wasn't sure when to use type aliases, and if I messed up anywhere, let me know!

### CODE REVIEW

File: src/um/oaidl.rs [hunk 1]
```diff
@@ -626,7 +626,69 @@ interface IDispatch(IDispatchVtbl): IUnknown(IUnknownVtbl) {
...
+    fn GetSize(
+        pcbSize: *mut ULONG,
+    ) -> HRESULT,
+    fn GetTypeInfo(
+        ppTypeInfo: *mut *mut ITypeInfo,
+    ) -> HRESULT,
+    fn GetField(
+        pvData: PVOID, 
+        szFieldName: LPCOLESTR, 
+        pvarField: LPVARIANT,
```
Comment (retep998) [inline src/um/oaidl.rs]: `*mut VARIANT`

File: src/um/oaidl.rs [hunk 2]
```diff
@@ -626,7 +626,69 @@ interface IDispatch(IDispatchVtbl): IUnknown(IUnknownVtbl) {
...
+    ) -> HRESULT,
+    fn GetFieldNoCopy(
+        pvData: PVOID, 
+        szFieldName: LPCOLESTR, 
+        pvarField: LPVARIANT,
```
Comment (retep998) [inline src/um/oaidl.rs]: `*mut VARIANT`

File: src/um/oaidl.rs [hunk 3]
```diff
@@ -626,7 +626,69 @@ interface IDispatch(IDispatchVtbl): IUnknown(IUnknownVtbl) {
...
+        pvData: PVOID, 
+        szFieldName: LPCOLESTR, 
+        pvarField: LPVARIANT, 
+        ppvDataCArray: *mut PVOID, 
+    ) -> HRESULT,
+    fn PutField(
+        wFlags: ULONG, 
+        pvData: PVOID, 
+        szFieldName: LPCOLESTR, 
+        pvarField: LPVARIANT,
```
Comment (retep998) [inline src/um/oaidl.rs]: `*mut VARIANT`

File: src/um/oaidl.rs [hunk 4]
```diff
@@ -626,7 +626,69 @@ interface IDispatch(IDispatchVtbl): IUnknown(IUnknownVtbl) {
...
+    ) -> HRESULT,
+    fn PutFieldNoCopy(
+        wFlags: ULONG, 
+        pvData: PVOID, 
+        szFieldName: LPCOLESTR, 
+        pvarField: LPVARIANT,
```
Comment (retep998) [inline src/um/oaidl.rs]: `*mut VARIANT`

File: src/um/oaidl.rs [hunk 5]
```diff
@@ -626,7 +626,69 @@ interface IDispatch(IDispatchVtbl): IUnknown(IUnknownVtbl) {
...
+    ) -> HRESULT,
+    fn GetFieldNames(
+        pcNames: *mut ULONG,
+        rgBstrNames: *mut BSTR,
+    ) -> HRESULT,
+    fn IsMatchingType(
+        pRecordInfo: *mut IRecordInfo, 
+    ) -> HRESULT,
```
Comment (retep998) [inline src/um/oaidl.rs]: This method returns `BOOL`

**Review by retep998 — CHANGES_REQUESTED** [review]

File: src/um/oaidl.rs (hunk 1 above)
Comment (ZerothLaw) [reply src/um/oaidl.rs]: Fixed. Thank you.

**Review by retep998 — APPROVED** [review]

**Pull request merged**